packages/peregrine/lib/Price/__tests__/__snapshots__/price.spec.js.snap

```diff
@@ -48,7 +48,7 @@ Array [
     1
   </span>,
   <span>
-     
+     
```

Comment:
Not sure why this changed. I just re-ran the tests and the snapshot was updated.

Reply:
Oh, I see you've already commented on this lol.